lib/order.rb

```diff
@@ -29,12 +31,13 @@ def show_basket
   end
```

Comment:
Very clever start method, welcoming customer/user to the restaurant, I never thought about it like that and potentially something I could absorb into my code.